app/src/components/MUIModal/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import MUIModal from '../MUIModal'
+import { styles } from './styles'
+import Button from '@material-ui/core/Button'
+import { withStyles, WithStyles } from '@material-ui/core/'
+
+interface ConfirmationModalProps extends WithStyles<typeof styles> {
+    confirmationText: string
+    handleClose: () => void
+    modalOpen: boolean
+    onConfirm: () => void
+    onCancel: () => void
+}
+
+function ConfirmationModal(props: ConfirmationModalProps) {
+    const {
+        modalOpen,
+        handleClose,
+        onConfirm,
+        onCancel,
+        confirmationText,
+        classes,
+    } = props
+
+    return (
+        <MUIModal handleClose={onCancel} modalOpen={modalOpen}>
```

Comment:
This takes care of my comment above, and makes the top-level handleClose prop obsolete